packages/react-storage/src/components/StorageManager/utils/uploadFile.ts

```diff
@@ -23,6 +24,7 @@ export function uploadFile({
   completeCallback,
   isResumable = false,
   provider,
+  metadata,
```

Comment:
If we do keep `metadata` we don't need to destructure it since the it will be part of the `rest` props

Reply:
```suggestion
```